bodhi/client/__init__.py

```diff
@@ -591,6 +591,11 @@ def edit(user: str, password: str, url: str, debug: bool, openid_api: str, **kwa
 @click.option('--packages', help='Query by package name(s)')
 @click.option('--content-type', help='Query updates based on content type',
               type=click.Choice(['rpm', 'module']))  # And someday, container.
+@click.option('--from-side-tag/--not-from-side-tag', default=None,
+              help='List only updates created from side-tag / not from side-tag')
+@click.option('--gating', help='Filter by gating status', default=None,
+              type=click.Choice(['passed', 'failed', 'ignored', 'waiting',
+                                 'running', 'queued', 'greenwave_failed']))
```

Comment:
I think we have removed the greenwave_failed status.

Reply:
Actually we still have that in the db since older update can still have that status, so we should have that value here :+1: